src/propelauth/token.rs

```diff
@@ -197,6 +197,7 @@ mod tests {
             first_name: Some("Easter".to_string()),
             last_name: Some("Egg".to_string()),
             username: None,
+            properties: None,
```

Comment:
`properties` is also declared on line 204 below. Can you remove this and the other 2 instances of adding `properties` to a `User` below?